src/Bicep.LangServer/Completions/BicepCompletionContextKind.cs

```diff
@@ -198,6 +198,11 @@ public enum BicepCompletionContextKind : ulong
         /// <summary>
         /// The current location is after the output type.
         /// </summary>
-        OutputTypeFollower = 1UL << 36
+        OutputTypeFollower = 1UL << 36,
+
+        /// <summary>
+        /// The current location is after the output type.
+        /// </summary>
+        OciModuleRegistryReference = ((ulong)1) << 37
```

Comment:
>((ulong)1)  [](http://example.com/codeflow?start=37&length=11)

nit - would `1UL` work here?

Reply:
Removed this as part moving logic to CompletionProvider:
https://github.com/Azure/bicep/commit/a699dbfc721de8ace5d95d935c541cfa51959a35